apps/small_filters/process.cpp

```diff
@@ -0,0 +1,109 @@
+#include <stdio.h>
+#include <memory.h>
+#include <assert.h>
+#include <stdlib.h>
+
+#include "../support/benchmark.h"
```

Comment:
You should be able to just

#include "benchmark.h"

If not, something about include paths is wrong. (Generally, prefer to avoid using .. in #includes.)